### PR TITLE
feat: add valhalla instruction to RoutingInstruction

### PR DESCRIPTION
**Pull request by farfromrefug**
It remove the need to format the instruction data.
It is much easier, takes the profile into account and supports language

### CODE REVIEW

**Comment by farfromrefug** [comment]
@mtehver what do you think about this? minimal change but big feature for valhalla

File: all/native/routing/OSRMRoutingProxy.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #ifndef _CARTO_OSRMROUTINGPROXY_H_
 #define _CARTO_OSRMROUTINGPROXY_H_
 
-#ifdef _CARTO_ROUTING_SUPPORT
+//#ifdef _CARTO_ROUTING_SUPPORT
```
Comment (mtehver) [inline all/native/routing/OSRMRoutingProxy.h:10]: I do not see why this should be for commented out

**Review by mtehver — COMMENTED** [review]
Sorry. Yes, I would like to merge this. I have just one comment, can you check this?

File: all/native/routing/OSRMRoutingProxy.h (hunk 1 above)
Comment (farfromrefug) [reply all/native/routing/OSRMRoutingProxy.h]: thanks fixed!

**Comment by mtehver** [comment]
Merged, thanks.

**Pull request merged**